### PR TITLE
Preparation for release

### PR DESCRIPTION
**Pull request by clechot**

### CODE REVIEW

**Comment by clechot** [comment]
Needs to be merged after PR #14

**Review by mistafunk — COMMENTED** [review]
Example 7 did not work for me on Windows, the paths in cell 3 are wrong. getcwd seems to depend where jupyter is started. So if we use venv activate the cwd of jupyter is inside the venv and it does not find the data folder anymore.

I created a new internal ticket to figure out how to properly read data files inside a jupyter workbook.

Also, I see some low prio issues on Linux:
* Chrome freezes for example 7 before I can click on anything... Firefox works though
* PyGEL3D is still not correctly packaged for Linux: the wheel suffix says "any" but then contains Windows dll and won't load on Linux. Can you maybe do a quick research if there is an alternative to PyGEL3D?

**Comment by clechot** [comment]
> Example 7 did not work for me on Windows, the paths in cell 3 are wrong. getcwd seems to depend where jupyter is started. So if we use venv activate the cwd of jupyter is inside the venv and it does not find the data folder anymore.

I don't think so. You should be in the examples root directory. On my machine, if I activate the environment from the root directory and then run `jupyter notebook`, I stay in the same folder.

File: envs/py36/requirements.txt [hunk 1]
```diff
@@ -65,7 +65,7 @@ prometheus-client==0.10.1
 prompt-toolkit==3.0.18
 ptyprocess==0.7.0
 pycparser==2.20
-PyGEL3D==0.0.27
+PyGEL3D==0.1.4
```
Comment (mistafunk) [inline envs/py36/requirements.txt:68]: This reverts commit 921ce717, i.e. we can remove toggling the version from the PR history.

File: ex7_building_modeling_optimization.ipynb [hunk 2]
```diff
@@ -65,7 +65,7 @@
     "import os\n",
     "from scipy import optimize\n",
     "import numpy as np\n",
-    "from PyGEL3D import gel, js"
+    "from pygel3d import hmesh, jupyter_display"
```
Comment (mistafunk) [inline ex7_building_modeling_optimization.ipynb:68]: This fixes PyGEL3D on Linux, nice!

The display has the wrong up axis, but at least we can see the model ;-)
image

**Review by mistafunk — CHANGES_REQUESTED** [review]
the commit message "updated package version to latest" is misleading - the version is not the fix but the change to hmesh and jupyter_display does the trick. please cleanup the commit history.

**Comment by mistafunk** [comment]
Cleaned up git history.

**Pull request merged**